zuul.d/jobs.yaml

```diff
@@ -1323,6 +1323,132 @@
     vars:
       ansible_collections_repo: github.com/ansible-collections/ansible.posix
 
+- job:
+    name: ansible-test-network-integration-base-ansible-utils
+    abstract: true
+    dependencies:
+      - name: build-ansible-collection
+        soft: true
+    pre-run:
+      - playbooks/ansible-test-base/pre.yaml
+      - playbooks/ansible-test-network-integration-base/pre.yaml
+    run: playbooks/ansible-test-base/run.yaml
+    post-run:
+      - playbooks/ansible-test-network-integration-base/post.yaml
+      - playbooks/ansible-test-base/post.yaml
+    required-projects:
+      - name: github.com/ansible/ansible
+      - name: github.com/ansible-collections/ansible.utils
+    timeout: 10800
+    vars:
+      ansible_collections_repo: github.com/ansible-collections/ansible.utils
+      ansible_test_command: network-integration
```

Comment:
We don't need network-integration test to be enabled for `ansible.utils` collection. The command I use to run the integration test for `ansible.utils` locally is for eg: `ansible-test integration -vvvv --python 3.8`